code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -55,10 +55,10 @@
 /mob/living/simple_animal/hostile/megafauna/ex_act(severity, target)
 	switch (severity)
 		if (1)
-			adjustBruteLoss(250)
+			adjustBruteLoss(400)
```

Comment:
I still dont understand the point of having a larger health pool if you're just going to scale the damage it takes to match it